### PR TITLE
vue-task

### PR DESCRIPTION
**Pull request by pavankumarhb**

### CODE REVIEW

**Comment by pavankumarhb** [comment]
Capture

File: one.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<head>
+    <title>Vue</title>
+</head>
+<body>
+   <div id='app'>
+      <h1 style="color: brown; padding:3%;">{{ header.toUpperCase() }}</h1>
```
Comment (santhoshgl) [inline one.html]: @janardhanakulakarni do not write inline styles css instead put in a class.

File: one.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      <img src="one.jpeg" style="padding-left: 0ch;"/><br><br>
+    
+      <input type="text" v-model="newitem" placeholder="add an items">
+      <button style="color:rgb(177, 71, 122)" :disabled="newitem.length===0" class="btn btn-primary" @click="save">save item</button><br>
+      <a href="https://www.justdial.com/Davangere/Vishal-Mega-Mart/9999P8192-8192-181128115038-B2Y5_BZDET">click here for more info</a>
+      <ul style="color:green; text-decoration: darkgreen;padding: 2%;">
```
Comment (santhoshgl) [inline one.html]: @janardhanakulakarni same here.

**Review by santhoshgl — CHANGES_REQUESTED** [review]

File: one.html (hunk 2 above)
Comment (pavankumarhb) [reply one.html]: I have created the separate css file please look at it.

**Pull request merged**